pokemon/src/Home.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import './App.css';
+import { Link } from 'react-router-dom';
+
+class Home extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: [],
+      isLoading: true,
+      hasError: false
+    }
+  }
+
+  componentDidMount() {
+    for (let i = 1; i <= 150; i++) {
```

Comment:
I think your loop led so some errors because of the high amount of fetch requests ;
index.js:1375 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
    in Home (created by Context.Consumer)

Reply:
To fetch more names in  one fetch use the following ;

https://pokeapi.co/api/v2/pokemon/?limit=60
this fetches 60 pokemon from the endpoint.